tests/test_exclusion_persistence.py

```diff
@@ -193,72 +191,10 @@ def test_connected_user(self):
             ep.import_(keep_unmentioned=True)
 
         result = User.query.get(owner_id)
-        self.assertEqual(result.email, 'sm-owner@gmail.com')
-        self.assertEqual(AuthProvider.query.count(), 1)
-        self.assertEqual(Token.query.count(), 1)
-        self.assertEqual(UserRelationship.query.count(), 1)
-
-    def test_preflight_valid(self):
```

Comment:
What happened to these test functions (can't tell if they were deleted or not...)? Were they duplicates?

Reply:
Yes, they were duplicates (see below them)